pages/marker.html

```diff
@@ -8,13 +8,64 @@
 
     <!-- import the webpage's stylesheet -->
     <link rel="stylesheet" href="../style.css" />
+    <link rel="stylesheet" href="../components/css/main.css" />
   </head>
   <body>
     <div class="header">
       <a href="../index.html">
-        <img src="img/logo.png" alt="logo" />
+        <img src="../assets/img/logo.png" alt="logo" />
         <p>AR.js Studio</p>
       </a>
     </div>
+
+    <div class="pages-content">
+      <a class="primary-button" href="../index.html">←</a>
+
+      <div class="pages-content-container">
+        <div class="pages-content-element">
+          <p class="title">Marker-based</p>
+          <p class="paragraph">
+            A marker-based AR experience uses an image to anchor the AR content.
+            Feel free to read the documentation, or follow the steps below to get started now.
+          </p>
+        </div>
+      </div>
+
+      <div class="pages-content-container">
+        <div class="pages-content-element">
+          <p class="lead">1. Upload an image to act as your marker</p>
+          <p class="paragraph">
+            Explain some constraints so people are not failing by uploading wrong images.
+          </p>
+          <a class="passive-button" href="#">Upload image</a>
+        </div>
+        <div class="pages-content-element">
+          <img src="https://dummyimage.com/380X380/d9d9d9/000000.png&text=Click+to+upload" alt="image placeholder" width="380">
```

Comment:
Added this dummy images just as placeholders, but I guess we can make a single div before user uploads content, let me know what do you think and I'll make the adjusments 😃

Reply:
I think it'd better to have a div w/ background, so it uses less bandwidth. Also, it'd have the same font style too.

Reply:
Yeah specs from figma:

**box:**

background: #F2F2F2;
border: 1px solid #D9D9D9;
box-sizing: border-box;
border-radius: 5px;

**Text:**

font-family: Chakra Petch;
font-style: normal;
font-weight: 600;
font-size: 19px;
line-height: 25px;
display: flex;
align-items: center;
color: #D9D9D9;

Reply:
Looks way better now 😄 

<img width="1049" alt="Screen Shot 2020-03-09 at 10 42 07 AM" src="https://user-images.githubusercontent.com/21111451/76231067-ad8e8580-61f2-11ea-86b9-a6890e52f606.png">